### PR TITLE
Felix's next topics

### PR DESCRIPTION
**Pull request by NolwennP**

### CODE REVIEW

File: content/whats-next/whats-next-mid-term.json [hunk 1]
```diff
@@ -17,6 +17,16 @@
   {
     "title": "Table attribute enhancements",
     "description": "We want to enrich our brand new Table attribute type with new columns types such as reference entities for example. We also want to add more options in the way your table completeness is computed.",
-    "areas": ["Governance", "Quality"]
+    "areas": ["Governance", "Data Quality"]
+  },
+  {
+    "title": "Tailored imports",
+    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations such as the ability to map options or format dates for example.",
```
Comment (NolwennP) [inline content/whats-next/whats-next-mid-term.json:24]:
```suggestion
    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations such as the ability to map options or to choose the dates format for example.",
```

File: content/whats-next/whats-next-mid-term.json [hunk 2]
```diff
@@ -17,6 +17,16 @@
   {
     "title": "Table attribute enhancements",
     "description": "We want to enrich our brand new Table attribute type with new columns types such as reference entities for example. We also want to add more options in the way your table completeness is computed.",
-    "areas": ["Governance", "Quality"]
+    "areas": ["Governance", "Data Quality"]
+  },
+  {
+    "title": "Tailored imports",
+    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations such as the ability to map options or to choose the dates format for example.",
```
Comment (felix-couprie) [inline content/whats-next/whats-next-mid-term.json:24]:
```suggestion
    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations (replacements, upper/lower case,date formats...)",
```

File: content/whats-next/whats-next-short-term.json [hunk 3]
```diff
@@ -5,5 +5,11 @@
     "img": "./img/table_attribute_templates.png",
     "areas": ["Governance", "Productivity"],
     "editions": ["GE","EE"]
+  },
+  {
+    "title": "Even more powerful tailored exports",
+    "description": "We will continue to deliver new features to enhance our tailored exports: product model support, CSV file format, ability to replace simple select values, and to clean the HTML tags of your text areas.",
+    "areas": ["Collaboration"],
+    "editions": ["GE","EE"]
```
Comment (felix-couprie) [inline content/whats-next/whats-next-short-term.json:13]: I suggest we get rid of that. Order associations is too big, too complex, and we will have to remake everything. For now, the topic is completely frozen.

**Review by felix-couprie — CHANGES_REQUESTED** [review]
Few comments

File: content/whats-next/whats-next-mid-term.json [hunk 4]
```diff
@@ -17,6 +17,16 @@
...
+  },
+  {
+    "title": "Tailored imports",
+    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations (replacements, first letters upper/lower case, date formats...)",
+    "areas": ["Collaboration", "Productivity"]
+  },
+  {
+    "title": "Association reorder",
+    "description": "We want to add the possibility to order your product associations.",
+    "areas": ["Governance", "Productivity"]
```
Comment (NolwennP) [inline content/whats-next/whats-next-mid-term.json:30]:
```suggestion
```

File: content/whats-next/whats-next-short-term.json (hunk 3 above)
Comment (NolwennP) [reply content/whats-next/whats-next-short-term.json:13]: You didn't comment at the right place, am I right?

**Review by felix-couprie — APPROVED** [review]
OK! Thanks

File: content/whats-next/whats-next-mid-term.json [hunk 5]
```diff
@@ -17,6 +17,11 @@
   {
     "title": "Table attribute enhancements",
     "description": "We want to enrich our brand new Table attribute type with new columns types such as reference entities for example. We also want to add more options in the way your table completeness is computed.",
-    "areas": ["Governance", "Quality"]
+    "areas": ["Governance", "Data Quality"]
+  },
+  {
+    "title": "Tailored imports",
```
Comment (camillebourjade) [inline content/whats-next/whats-next-mid-term.json:23]:
```suggestion
    "title": "Tailored Imports",
```

File: content/whats-next/whats-next-mid-term.json [hunk 6]
```diff
@@ -17,6 +17,11 @@
...
+    "description": "We want you to be able to import simple flat files with formats differing from ours. In this first step, we want to allow various basic data transformations (replacements, first letters upper/lower case, date formats...)",
```
Comment (camillebourjade) [inline content/whats-next/whats-next-mid-term.json:24]:
```suggestion
    "description": "We want you to be able to import simple flat files in your own formats. In this first step, we want to allow various basic data transformations (replacements, first letters upper/lower case, date formats...)",
```

**Review by camillebourjade — APPROVED** [review]
English fix

**Pull request merged**